lib/has_scope/version.rb

```diff
@@ -1,3 +1,3 @@
 module HasScope
-  VERSION = "0.7.0"
+  VERSION = '0.8.0'.freeze
```

Comment:
Can you avoid bumping the gem and the adding a changelog header? I don't think we are planning to release a new minor release